### ISSUE TITLE
Fix: avoid resolve promise with empty argument

### ISSUE THREAD

**Issue by davikawasaki** [pixelygroup/keycloak-koa-connect#15]
# Context

I'm developing a typescript KoaJS application that needs to connect with this keycloak connector to protect routes. Whenever I run `tsc`, the following error appears below:

![image](https://user-images.githubusercontent.com/12398127/120424811-78b0a600-c343-11eb-9c23-737c49d687f6.png)

The `tsconfig.json` file can be found written as below:

```json
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es2018",
        "noImplicitAny": false,
        "outDir": "lib",
        "rootDir": "src/main",
        "sourceMap": true,
        "strictNullChecks": false
    },
    "include": [
        "./src/**/*",
        "./config/**/*",
        "node_modules/node-config-ts/global.d.ts"
    ],
    "exclude": [
        "node_modules"
    ]
}
```

# Solution

Add a parameter to this `resolve` with a `true` value. This way it doesn't block any `tsc` process. Solution can be found at PR #16.

**Comment by davikawasaki** [comment]
Continuation at: https://github.com/pixelygroup/keycloak-koa-connect/pull/16

**Comment by davikawasaki** [comment]
Resolved on #16.
